web/tellus_app/api/tests/authzsetup.py

```diff
@@ -1,9 +1,9 @@
 import time
```

Comment:
Kan deze code niet naar de auth library?

Reply:
Goed punt! Een deel wel denk ik. Maar je moet nog wel in het project zelf definieren wat er in het token moet staan.